std_msgs/Header header
int64 vel_ref     # thrust_ref reading

